DSP_API/ThumbDV/thumbDV.c

```diff
@@ -277,34 +285,38 @@ void thumbDV_dump( char * text, unsigned char * data, unsigned int length ) {
     }
 }
 
-static void thumbDV_writeSerial( FT_HANDLE handle , unsigned char * buffer, uint32 bytes )
+static int thumbDV_writeSerial( FT_HANDLE handle , unsigned char * buffer, uint32 bytes )
 {
     FT_STATUS status = FT_OK;
     DWORD written = 0;
 
     if ( handle != NULL )
     {
+    	//FT_SetRts(handle);
         status = FT_Write(handle, buffer, bytes, &written);
+        usleep(10000);
```

Comment:
Is this delay supposed to stay in the production code?

Reply:
This delay allows for the TX to be demodulated on the faster baud. It is up to you guys if you want to keep it. Really isn't a solid fix

Reply:
If the faster baud rate doesn't work, what is the value of leaving this in?

Reply:
Took this out